code/modules/admin/admin_verbs.dm

```diff
@@ -406,8 +406,14 @@ var/list/admin_verbs_mentor = list(
 	set category = "Event"
 	set name = "Give Spell"
 	set desc = "Gives a spell to a mob."
-	var/obj/effect/proc_holder/spell/wizard/S = input("Choose the spell to give to that guy", "ABRAKADABRA") as null|anything in spells
-	if(!S) return
+	var/list/spell_list = list()
+	var/type_length = length("/obj/effect/proc_holder/spell") + 2

```

Comment:
Spoke to @tigercat2000 about this one, originally had it be `/obj/effect/proc_holder/spell/wizard`, but changed it back in case there's some spell that I missed that doesn't have the /wizard/ subtype.